.github/workflows/load-test.yaml

```diff
@@ -0,0 +1,141 @@
+name: On-demand load test
+
+on:
+  workflow_dispatch:
+    inputs:
+      ref:
+        required: true
+        description: SHA or branch name to deploy
+        type: string
+        default: 'main'
+      N:
+        required: true
+        description: large N will generate more tuples
+        type: number
+        default: 1
+      M:
+        required: true
+        description: large M implies a single user has a relationship with multiple objects
+        type: number
+        default: 1
+      K:
+        required: true
+        description: large K implies a single object has a relationship with multiple users
+        type: number
+        default: 1
+      testCase:
+        required: true
+        description: the test case to run (see https://github.com/openfga/benchmarks)
+        type: choice
+        options:
```

Comment:
Do we want to omit options so that if we don't need to update this file if there are more options later on?

Reply:
For now i'd rather keep as-is so that i don't have to figure out what test cases we have. Later on if we see that this list becomes unmanagable/frequently updated, we can change this.